protos/browser.proto

```diff
@@ -16,13 +16,10 @@ syntax = "proto3";
 
 package wptdashboard;
 
-// Browser holds objects that appear in browsers.json
-message Browser {
-  bool initially_loaded = 1;
-  bool currently_run = 2;
-  string browser_name = 3;
-  string browser_version = 4;
-  string os_name = 5;
-  string os_version = 6;
-  bool sauce = 7;
+enum Browser {
+  BROWSER_UNKNOWN = 0;
+  CHROME = 1;
+  EDGE = 2;
+  FIREFOX = 3;
+  SAFARI = 4;
```

Comment:
```go
enum Browser {
  UNKNOWN = 0;
  CHROME = 1;
  EDGE = 2;
  FIREFOX = 3;
  SAFARI = 4;
}
```

WDYT?

Reply:
Without putting these in a different package, we can only use `UNKNOWN` once in an enum (the names get hoisted up into their package namespace a-la C/C++). I wish it weren't so; I would tend to agree with you.

Reply:
> we can only use UNKNOWN once in an enum...
> ... (the names get hoisted up into their package namespace a-la C/C++

Well, at least it's consistent with _something_ ;) Thanks for the follow up

Reply:
From the Googleverse somewhere:
> The recommendation is to prefix the enum values with the enum’s name if there is a chance of name clashes

Ugly, but there it is